files/galaxy/config/user_preferences_extra_conf.yml

```diff
@@ -1,246 +1,274 @@
-      Bring your own Pulsar endpoint to Galaxy. You can add here your Pulsar credentials and specifications.
-      After 24 hours Galaxy's job scheduling systems will take your Pulsar into account and schedule appropriate jobs to your compute resources.
-      This is an experimental feature. Contact us if you want to learn more about it.
-    inputs:
-      - name: username
-        label: RabbitMQ custom username (e.g. mypulsarendpoint).
-        type: text
-        required: False
-      - name: password
-        label: RabbitMQ custom alphanumerical password, at least 10 characters long (e.g. MyPulS4rP4ssw0rd).
-        type: password
-        required: False
-      - name: max_accepted_cores
-        label: Maximum number of CPU cores available.
-        type: number
-        required: False
-      - name: max_accepted_mem
-        label: Maximum number of RAM available (in GB).
-        type: number
-        required: False
-      - name: min_accepted_gpus
-        label: Minimum number of GPU available (set this to zero if you don't want to share GPUs).
-        type: number
-        required: False
-      - name: max_accepted_gpus
-        label: Maximum number of GPU available (set this to zero if you don't want to share GPUs).
-        type: number
-        required: False
-
-  omero_account:
-    description: Your OMERO instance connection details
-    inputs:
-      - name: username
-        label: Username
-        type: text
-        required: False
-      - name: password
-        label: Password
-        type: password
-        required: False
-
-  ena_webin_account:
-    description: Your ENA Webin account details
-    inputs:
-      - name: webin_id
-        label: ENA Webin ID
-        type: text
-        required: False
-      - name: password
-        label: Password
-        type: password
-        required: False
-
-  c3s_account:
-    description: Your CDS API Key (Copernicus Climate Change Service API Key)
-    inputs:
-      - name: c3s_cds_apikey
-        label: C3S CDS API Key
-        type: password
-        required: False
-
-  cads_account:
-    description: Your Copernicus ADS API Key (Copernicus Atmosphere Data Store API Key)
-    inputs:
-      - name: cads_apikey
-        label: Copernicus ADS API Key
-        type: password
-        required: False
-
-  cmems_account:
-    description: Your Copernicus CMEMS API Key (Copernicus Marine Service API Key)
-    inputs:
-      - name: cmems_username
-        label: Copernicus CMEMS username
-        type: text
-        required: False
-      - name: cmems_password
-        label: Password
-        type: password
-        required: False
-
-  gitlabfs:
-    description: Your DataPLANT ARC Gitlab access details https://git.nfdi4plants.org
-    inputs:
-      - name: read_access_token
-        label: API Read Access Token
-        type: password
-        required: False
+    webhook_overlay_search:
+        description: Configuration option for your search results
```

Comment:
is 4 spaces indentation intentionally or by accident?

Reply:
intentionally, the upstream file is like that, not sure why.

Reply:
in this case we need to be careful in the future